src/webui/www/private/scripts/contextmenu.js

```diff
@@ -169,24 +169,30 @@ window.qBittorrent.ContextMenu = (function() {
             }.bind(this));
 
             elem.addEvent('touchstart', function(e) {
-                e.preventDefault();
-                clearTimeout(this.touchstartTimer);
                 this.hide();
-
-                const touchstartEvent = e;
-                this.touchstartTimer = setTimeout(function() {
-                    this.touchstartTimer = -1;
-                    this.triggerMenu(touchstartEvent, elem);
-                }.bind(this), this.options.touchTimer);
+                this.touchStartAt = new Date();
```

Comment:
https://stackoverflow.com/a/15641427

Reply:
I wasn't aware of this API - thanks for pointing it out!